.storybook/stories/DataFiltering/components/DashboardPagination.tsx

```diff
@@ -24,8 +24,37 @@ export const DashboardPagination = () => {
 			})}
 			itemsPerPage={pagination.perPage}
 			onItemsPerPageChange={(perPage) =>
-				setPagination({ ...pagination, perPage })
+				setPagination({
```

Comment:
![pagination-re-render](https://github.com/agriculturegovau/agds-next/assets/853552/88ffe951-36f4-4568-8549-b39052e56461)
Now that it's not broken, it's highlighted that the pagination behaves weirdly and renders in-between states. Is there anything we can do here?

Reply:
Good spot! I wonder if that's the mock load time I saw. I'll investigate.

- [ ] find and fix the in-between pagination states